task/repository/start_task.go

```diff
@@ -0,0 +1,66 @@
+package repository
+
+import (
+	"context"
+	"database/sql"
+	"kodiiing/task"
+	"time"
+
+	"github.com/jackc/pgx/v5"
+)
+
+type StartTaskOut struct {
+	Task
+
+	Completed         bool
+	CompletedAt       sql.NullTime
+	SatisfactionLevel int64
+}
+
+func (r *Repository) StartTask(ctx context.Context, userId, taskId int64) (out StartTaskOut, err error) {
+	if taskId == 0 || userId == 0 {
```

Comment:
check db masih idup atau ga xD

Reply:
usernya atau task nya? atau keduanya?. Awalnya aku mau cek is user exists, tapi di schema db udah ada constraint `references`

Reply:
Punten...